hello-bridge/README.md

```diff
@@ -253,6 +242,14 @@ be used by teams of DApp builders who will handle data provisioning on behalf of
 Such teams will want to define and deploy their own contracts as shown in the [custom contract
 bridging] tutorial.
 
+## Deploying Your Own Contract
```

Comment:
This kind of steals the purpose of the next tutorial `custom-contracts-bridging`. That tutorial extends hello bridge by having the user deploy their own contracts.

We have two good options:
1. Get rid of the whole `custom-contracts-bridging` tutorial in favor of DEPLOY.md
2. Move DEPLOY.md into the `custom-contracts-bridging` tutorial folder and refer to it there. Hello-bridge would continue to exclusively rely on pre-established contracts.